gluster-exporter/metric_volume.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	volumeHealLabels = []MetricLabel{
+		{
+			Name: "volume",
+			Help: "Volume Name",
+		},
+		{
+			Name: "brick_path",
+			Help: "Brick Path",
+		},
+		{
+			Name: "host",
+			Help: "Hostname",
```

Comment:
Host FQDN or IP?

Reply:
IP addr. should I change the label here?.

Reply:
All other places we saying something like 

```
                {
                        Name: "host",
                        Help: "Host name or IP",
                },
```